collectors/pool_usage.go

```diff
@@ -195,6 +223,8 @@ type cephPoolStats struct {
 			ReadBytes    float64 `json:"rd_bytes"`
 			WriteIO      float64 `json:"wr"`
 			WriteBytes   float64 `json:"wr_bytes"`
+			QuotaMaxBytes   float64 `json:"quota_bytes"`
```

Comment:
Possible to run `gofmt` on this? Also it would help to go through https://github.com/digitalocean/ceph_exporter/blob/master/CONTRIBUTING.md.